BE/baseball/src/main/java/com/codesquad/baseball/controller/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.codesquad.baseball.controller;
+
+import com.codesquad.baseball.error.ErrorResponse;
+import com.codesquad.baseball.error.exception.NotFoundException;
+import com.codesquad.baseball.error.exception.TeamNotPlayableException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(NotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleNotFoundException(NotFoundException e) {
+        ErrorResponse response = ErrorResponse.of(e.getErrorCode());
+        return new ResponseEntity<>(response, response.getStatus());
```

Comment:
지역 변수의 할당이 불필요할 땐 할당을 최소화하는 쪽으로 개선해보시는 것도 좋습니다.

```suggestion
        return new ResponseEntity<>(ErrorResponse.of(e.getErrorCode()), response.getStatus());

Reply:
감사합니다. 수정하겠습니다.